Pinta.Core/Managers/WorkspaceManager.cs

```diff
@@ -500,7 +493,7 @@ private Task ShowFilePermissionErrorDialog (
 	public event EventHandler? SelectedLayerChanged;
 	public event PropertyChangedEventHandler? LayerPropertyChanged;
 
-	public event EventHandler<DocumentEventArgs>? DocumentCreated;
+	public event EventHandler<DocumentEventArgs>? DocumentAttached;
```

Comment:
Same as above, this should match the updated name e.g. `DocumentAdded`

I think `DocumentOpened` underneath is also unused, so please remove that too to avoid any confusion

Reply:
That's true, this event is not used, so I removed it